rpc-rs/src/provider_main.rs

```diff
@@ -27,6 +27,25 @@ const DEFAULT_NATS_ADDR: &str = "0.0.0.0:4222";
 
 /// Start provider services: tokio runtime, logger, nats, and rpc subscriptions
 pub fn provider_main<P>(provider_dispatch: P) -> Result<(), Box<dyn std::error::Error>>
+where
+    P: ProviderDispatch + Send + Sync + Clone + 'static,
+{
+    // get lattice configuration from host
+    let host_data = match load_host_data() {
```

Comment:
Per [this issue](https://github.com/wasmCloud/wasmcloud-otp/issues/219), I think the idea was that we were going to go with a `config_json` field that was nothing but an opaque string to the OTP runtime host, and could be interpreted any way the provider likes.

Note that the control interface already supports this field as an opaque string, as you can see here in [this smithy](https://github.com/wasmCloud/interfaces/blob/main/lattice-control/lattice-control-interface.smithy#L376).

The runtime OTP host _does not currently_ support the config json field, but it can in a PR we create today.

Reply:
ok. Changing the field name requires updating the smithy file which is out of scope for today because it isn't required for demos. Will change this in a later PR.